.travis.yml

```
@@ -24,54 +21,27 @@ jobs:
         - chmod +x ./scripts/get-updated-distros.sh
         - ./scripts/check-asciidoctor-build.sh || travis_terminate 1
         - |
-          ./scripts/get-updated-distros.sh |
-            while read -r filename; do
-              if [ "$filename" == "_topic_maps/_topic_map.yml" ]; then python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.14 --no-upstream-fetch
-
-              elif [ "$filename" == "_topic_maps/_topic_map_osd.yml" ]; then python3 build.py --distro openshift-dedicated --product "OpenShift Dedicated" --version 4 --no-upstream-fetch
-
-              elif [ "$filename" == "_topic_maps/_topic_map_ms.yml" ]; then python3 build.py --distro microshift --product "Microshift" --version 4 --no-upstream-fetch
-
-              elif [ "$filename" == "_topic_maps/_topic_map_rosa.yml" ]; then python3 build.py --distro openshift-rosa --product "Red Hat OpenShift Service on AWS" --version 4 --no-upstream-fetch
-
-              elif [ "$filename" == "_distro_map.yml" ]; then python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.14 --no-upstream-fetch
-              fi
-            done
+          ./scripts/get-updated-distros.sh | while read -r filename; do
+            case "$filename" in
+              "_topic_maps/_topic_map.yml")
+                python3 build.py --distro openshift-enterprise --product 'OpenShift Container Platform' --version 4.14 --no-upstream-fetch
```

```suggestion
                python3 build.py --distro openshift-enterprise --product 'OpenShift Container Platform' --version 4.14 --no-upstream-fetch
```

It would be great if we could add a variable for version that is picked up from somewhere so we could have a fully portable build script that does not need modification between branches. Nice to have maybe.

---

I'm reverting the changes with switch case, because there might be a module that appers in multiple distros and switch case will only run for single instance and exit.